app/templates/thank-you.html

```diff
@@ -4,6 +4,22 @@
 
 {% block survey %}
 
-  <p>Thank you for submitting the survey {% if data.submitted_at %} on {{data.submitted_at}} {% endif %}</p>
+  <div class="header u-mb-m u-mb-l@s">
+    <h1 class="header__title">Thank You</h1>
+    <div>Monthly Business Survey - Retail Sales Index</div>
+    <div class="u-lighter">Period: 4th January 2016 to 31st January 2016</div>
```

Comment:
Having talked about it, this isn't easily achieved. @alexmorris to review wether we need it.